src/product-info.js

```diff
@@ -42,5 +42,9 @@ export default {
   'user-rhsso': {
     prettyName: 'Red Hat Single Sign-On',
     gaStatus: 'GA'
+  },
+  ups: {
+    prettyName: 'UnifiedPush Server',
+    gaStatus: 'limited availability'
```

Comment:
Adding UPS product info here as this is now installed by the Integreatly operator. 

As far as I know the gaStatus for this should be `limited availability` but I'm currently still confirming this with the ups team. @tiffanynolan @mfrances17 Is there any preference on how to display this information? Should it be `LA` or `limited availability`?

Reply:
Removed this, will be included in a follow up task if required. https://issues.jboss.org/browse/INTLY-3787